awx/main/models/ha.py

```diff
@@ -139,37 +143,12 @@ def is_lost(self, ref_time=None, isolated=False):
     def is_controller(self):
         return Instance.objects.filter(rampart_groups__controller__instances=self).exists()
 
+    def is_receptor(self):
+        return self.version.startswith('ansible-runner-')
+
     def is_isolated(self):
         return self.rampart_groups.filter(controller__isnull=False).exists()
 
-    def refresh_capacity(self):
```

Comment:
Capacity is no longer calculated by this code via `psutil`, but is instead accomplished via an adhoc command that is run on every new `Instance` when it joins the receptor network via Ansible fact gathering:

https://github.com/ansible/awx/pull/9879/files#diff-6fb7bd98930ec6a05c1ae9296b41b63921e1813dcee53895f16769f910917c38R3030-R3041